gprofiler/metadata/system_metadata.py

```diff
@@ -205,11 +242,11 @@ def get_hostname() -> str:
 def _initialize_system_info() -> Any:
     # initialized first
     global hostname
-    hostname = "<unknown>"
-    distribution = ("unknown", "unknown", "unknown")
-    libc_version = ("unknown", "unknown")
-    mac_address = "unknown"
-    local_ip = "unknown"
+    hostname = f"<{UNKNOWN_VALUE}>"  # < > are added to further distinct it from a legit hostname
```

Comment:
Set it to `None` until we obtain it

Reply:
All others are None as well. Not making this change now